lib/blockscore/base.rb

```diff
@@ -92,6 +85,13 @@ def add_setter(symbol, *_args)
 
     private
 
+    def force!
+      res = @proc.call
```

Comment:
I wouldn't bother assigning to an lvar here and just use it inline on the line below.